docs/RELEASE.md

```diff
@@ -0,0 +1,126 @@
+`polkadot/node/primitives/src/lib.rs`.
+
+### Westend & Rococo
+
+For the these networks, we only increment the `spec_version` and sometimes the `transaction_version`. The spec version is also following
+the node version. Its schema is: `M_mmm_ppp` and for example `1_002_000` is the node release `1.2.0`. This versioning has no further meaning, and is only done to map from an on chain `spec_version` easily to the release in this repository. 
+
+## Backports
+
+Backports should most of the time not be required. We should only backport [critical bug fixes](#bug-and-security-fix) and then release the fixed crates. There should be no need to backport anything from a release branch.
+
+# Processes
+
+The following processes are necessary to actualize our releases. Each process has a *Cadence* on which it must execute and an *Responsible* that is responsible for autonomously doing so and reporting back any error in the RelEng<sup>1</sup> channel.
+
+## Crate Bumping
+
+Cadence: (possibly) each Merge Request. Responsible: Developer that opened the MR.
+
+Following SemVer isn't easy, but there exists [a guide](https://doc.rust-lang.org/cargo/reference/semver.html) in the Rust documentation that explains the small details on when to bump what. This process should be augmented with CI checks that utilize [`cargo-semver-checks`](https://github.com/obi1kenobi/cargo-semver-checks) and/or [`cargo-public-api`](https://github.com/Enselic/cargo-public-api). They must also pay attention to downstream dependencies that require a version bump, because they export the changed API.
+
+### Steps
+
+1. Developer opens a Merge Request with changed crates against `master`.
+2. They bump all changed crates according to SemVer.
+3. They bump all crates that export any changed types in their *public API*.
+4. They also bump all crates that inherit logic changes from relying on one of the bumped crates. 
+
+## Mainline Release
+
+Cadence: every two weeks. Responsible: Release Team.
+
+This process aims to release the `release` branch as a *Mainline* release every two weeks. It should eventually be automated.
+
+### Steps
+
+1. Check if process [Clobbering](#clobbering) needs to happen and do so, if that is the case.
+2. Check out the latest commit of `release`.
+3. Verify all CI checks of that commit.
+4. Update the `CHANGELOG.md` version and date. <!-- Here we could aggregate the PrDocs into the CHANGELOG -->
+5. Open a Merge Request against `release` for visibility.
+6. Check if there were any changes since the last release and abort, if not.
+7. Run `cargo semver-checks` and `cargo public-api` again to ensure that there are no SemVer breaks.
+8. Internal QA from the release team can happen here.
+9.  Do a dry-run release to ensure that it *should* work.
+10. Merge it into `release`.
+11. Verify all CI checks.
+12. Comment that a *Mainline* release will happen from the merged commit hash.
+13. Release all changed crates to crates.io.
+14. Create a release on GitHub.
```

Comment:
100%!! All of this should be done by some bot. Only the final merge should be  done by a human. Dry run of a release would just be CI `cargo build --release` and then run all checks. We should also potentially add more checks as required.

Reply:
Yea sure. That is why i wrote `It should eventually be automated`. All of this can be automated.  

With dry run i meant `cargo release --dry-run` for all crates in the correct order. Not sure if this is possibly without mocking crates-io (like JP did) but i hope so.